bower.json

```diff
@@ -3,7 +3,5 @@
   "version": "0.0.0",
   "directory": "client/bower_components",
   "dependencies": {
```

Comment:
agree with these changes but feben already did them on her branch so this will create a merge conflict. going forward please don't touch the client folder w/o coordinating w/ feben.